proteus/LinearSolvers.py

```diff
@@ -514,6 +517,7 @@ def prepare(self,b=None):
             self.petsc_L.setValuesLocalCSR(self.csr_rep[0],self.csr_rep[1],self.csr_rep[2],p4pyPETSc.InsertMode.ADD_VALUES)
         self.petsc_L.assemblyBegin()
         self.petsc_L.assemblyEnd()
+        self.petsc_L.save("L")
```

Comment:
We should probably comment this out for now or add a `--print-matrices` option that defaults  to `False`